woof-distro/x86_64/debian/bullseye64/DISTRO_PKGS_SPECS-debian-bullseye

```diff
@@ -618,6 +618,7 @@ yes|xclip|xclip|exe,dev,doc,nls
 no|xcur2png||exe #pcur needs this
 no|xdelta||exe
 no|xdg-puppy-jwm||exe
+yes|xdg-utils|xdg-utils|exe,dev,doc,nls
```

Comment:
I did it this way in raspup. Is there any advantage in your way?

```
yes|xdg-utils|xdg-utils|exe>null,dev>null,doc>null,nls>null #fake install
```












'

Reply:
xdg-utils contains more than just xdg-open. I considered a "fake install", but decided to fix this issue without breaking stuff that needs the other executables in xdg-utils.